tests/page-objects/dashboard/DashboardPage.ts

```diff
@@ -177,6 +184,36 @@ class DashboardPage extends BasePage {
         ]);
     }
 
+    async duplicateDashboardFromWorkbook(dashId: string, newDashName: string) {
+        const workbookPO = new Workbook(this.page);
+
+        await workbookPO.openE2EWorkbookPage();
+
+        await workbookPO.openWorkbookItemMenu(dashId);
+
+        await this.page.locator(slct(WorkbookPage.MenuItemDuplicate)).click();
+
+        // waiting for the dialog to open, specify the name, save
+        await this.workbookDialogFillAndSave(this.page, newDashName);
+
+        await this.page
+            .locator(`${slct(WorkbookPage.ListItem)}:has-text('${newDashName}')`)
+            .click();
+    }
+
+    // Fill in the input with the name of the entity being created in the EntryDialog (the dialog that appears when saving entities) and click the "Create" button
+    async workbookDialogFillAndSave(page: Page, entryName: string) {
```

Comment:
There is a similar function here https://github.com/datalens-tech/datalens-ui/blob/main/tests/page-objects/workbook/DialogCreateEntry.ts#L28